src/scss/parts/_header.scss

```diff
@@ -7,9 +7,13 @@
 // Général design
 //
 header#pm_header-desktop {
-	background: $color-header;
-	border-bottom: 1px solid darken($color-header, 5%);
-	z-index: 100;
+ background: $color-header;
+ /* Rectangle: */
+ background-image: -webkit-linear-gradient(bottom, $color-header, #46477F);
```

Comment:
I prefer to stick with flat header for now... I'll add  an option to modify header color or gradient from the main scss file soon. BTW you don't need to specify browser prefixes, they are handled by Gulp/PostCSS.